locales/en.default.schema.json

```diff
@@ -958,7 +958,21 @@
           "options__3": {
             "label": "Large"
           },
-          "label": "Image ratio"
+          "label": "Image height",
+          "info": "Determines section height."
```

Comment:
This is expected, and it already behaves this way. It's based on the image aspect ratio. If we always want the full width of the image to be visible, then the entire section would need to be revisited because the section's behaviour (no-overlap) is based on the image being the same height as the text. I'll leave this as is, and talk to Wik about this as a follow up.

Reply:
Ah sounds good 👍